gspread/worksheet.py

```diff
@@ -2330,10 +2434,12 @@ def copy_to(
             the newly created sheet.
         :rtype: dict
         """
-        return self.client.spreadsheets_sheets_copy_to(spreadsheet_id, self.id)
+        return self.client.spreadsheets_sheets_copy_to(
+            self.spreadsheet_id, self.id, destination_spreadsheet_id
+        )
 
     @cast_to_a1_notation
-    def merge_cells(self, name, merge_type="MERGE_ALL"):
+    def merge_cells(self, name: str, merge_type: str = "MERGE_ALL") -> JSONAnyType:
```

Comment:
it seems as if merge_type could be an Enum, if there are only 3 options.

Reply:
sounds good :) we can resolve this conversation when issue created

Reply:
done see #1259